cypress/integration/save.cy.js

```diff
@@ -184,21 +185,36 @@ describe('saving an AO', () => {
         })
     })
 
-    /*
-    describe('"save" for a saved AO created by others', () => {
-        it('navigates to the start page', () => {
+    // Specific test for DHIS2-15722
```

Comment:
you shouldn't really have to make this comment. The `describe` should include sufficient info to know what's being tested.